### ISSUE TITLE
Course play not stopping when items are in the way.

### ISSUE THREAD

**Issue by johnonline1** [Courseplay/courseplay#3782]
If there is an item / vehicle in the way of the course-play vehicle it hits it and continues to push the item out of the way.

**Comment by Tensuko** [comment]
Thats currently the state...
Collision detection doesn't work for everything yet.
Vehicle detection is something different.

**Comment by johnonline1** [comment]
Both need to be looked at if you can please.

**Comment by lcc1211** [comment]
Good luck with this. We reported this right after FS17 came out and it was immediately placed on a postponed status and then closed after FS19 came out.

**Comment by Tensuko** [comment]
So you want to tell me what ? That you got time and a Idea to solve this and bring it into CP ?

**Comment by lcc1211** [comment]
No. Just giving fair warning that this was either an issue that was too difficult to fix or there was no interest in fixing it. I don't know which since it was almost immediately placed in a postponed status - just like this one - and after a bit no other replies were ever added to it.

**Comment by Tensuko** [comment]
This is simply not our priority number one, there are more important things then this. I know that this is needed too, but before mode2 and some fixes are not done, we wont start a project like this. Postponed doesnt mean ignored..... Pls be just patient, I will discuss this with the team

**Comment by Marisolaa** [comment]
I do hope this becomes a priority.  Especially difficult on small maps, vehicles take off for parts unknown.  Reset to shop can be a long way off.  When getting more than 4 fields, especially on crops that need all 3 vehicles running plus offloading, I've spent more time fixing traffic lately.  Farther fields struggle as well, as offloading tractors start stacking up and smashing each other at  silos, making you choose to add a second dumping vehicle to the plan (creating even more traffic) or babysitting the silo.  Or, conversely, trying to set up multiple silos and manage runs silo to silo to silo.  The vanilla maps are brutal to semis, compounding the issue.

After a few fields, it actually becomes "gamebreaking" in using Courseplay.  I've not even been able to stop and care for animals or make silage, even though I've chopped my fields in close to avoid getting onto the road, backing up on the field only compounds the problem.  The offloading vehicles end up crashing into trees, seeders, the harvester - all of it.  By the time I unscramble those, another field is crashing.
